### DISCUSSION TITLE
use prettier on json files

### DISCUSSION THREAD

**Discussion by onemen** [epicweb-dev/epic-stack#159] [Ideas]
we can use prettier on json files and even on .prettierrc files by using [overrides](https://prettier.io/docs/en/configuration.html#configuration-overrides)

```diff
	useTabs: true,
+	overrides: [
+		{
+			files: ['**/*.json'],
+			options: {
+				useTabs: false,
+				parser: 'json',
+			},
+		},
+	],
```

**Comment by MichaelDeBoey** [comment]
As you can see in `package.json`, Prettier is run on all files
https://github.com/epicweb-dev/epic-stack/blob/9d02e65ca841693c02a335e6bd674252d2f93aa3/package.json#L15

Prettier can auto-detect the file-format & choose the needed parser itself, so no need to be explicit about it

**Reply by onemen** [reply]
OK,

but we still need this override to prettify json files with `useTabs: false`

**Reply by kentcdodds** [reply]
I hadn't thought of that, but you're right

**Reply by kentcdodds** [reply]
And if we do that then we can un-ignore package.json too which would be nice
